UnityProject/Assets/Scripts/HealthV2/Living/PolymorphicSystems/ReagentPoolSystem.cs

```diff
@@ -33,6 +33,7 @@ public class ReagentPoolSystem : HealthSystemBase
 		public void AddFreshBlood(ReagentMix bloodPool, float amount)
 		{
 			// Currently only does blood and required reagents, should at nutriments and other common gases
+			if(bloodPool == null || bloodType == null || bloodType.CirculatedReagent == null) return;
```

Comment:
This is because you haven't it setup initial system on the mob

Reply:
Well I don't want to, but It's trying to set it up by itself even though I did not add any systems to it.
so the quickest solution to me was to just check if anything was null and not block the entire foreach loop for initializing because one rouge system

Reply:
is because the brain has  blood circulated component, and so that Generates the system

Reply:
removed the return and added an error to remind people about defining systems